src/V3Width.cpp

```diff
@@ -2156,8 +2156,21 @@ class WidthVisitor final : public VNVisitor {
                 UINFO(9, "Unsized becomes dynamic array " << nodep << endl);
                 AstDynArrayDType* const newp
                     = new AstDynArrayDType{unsizedp->fileline(), unsizedp->subDTypep()};
```

Comment:
This code is similar to below, so probably can reorganize the loop to move this code inside the loop?

Reply:
I will see into it. Also it looks like this needs to be modified to support unsized dimensions mixed with sized ones.

Reply:
Makes sense.  Most likely to get all data types working the code will need to recurse into other visitors here based on what's inside.